package com.jjobkorea.dto;

import lombok.Data;

@Data
public class EmailDTO {
	// 이메일 주소
    private String mail;
    // 인증 코드
    private String verifyCode;
}